packages/test/test-utils/src/testObjectProvider.ts

```diff
@@ -235,11 +237,15 @@ export class TestObjectProvider {
      */
     public async createContainer(entryPoint: fluidEntryPoint, options?: ITestLoaderOptions) {
         const loader = this.createLoader([[defaultCodeDetails, entryPoint]], options);
+        if (this._usedDocumentId) {
+            throw new Error("createContainer can only be called once! Use loadContainer");
```

Comment:
The error should probably say only a single container / document can be created. To load the container / document use loadContainer. Similar for the other place where this error is thrown.